.travis.yml

```diff
@@ -39,10 +39,14 @@ before_install:
 install:
     # Create the python environment ;
     - conda create -q -n testenv python=$TRAVIS_PYTHON_VERSION numpy pip cython matplotlib;
+    - conda install pyqt
     - source activate testenv
     # Testing dependencies
     - pip install codecov pytest pytest-cov;
     - pip install neuropycon_data
+    - pip install git+https://github.com/vispy/vispy.git
+    - pip install pandas PyOpenGL PyOpenGL_accelerate
+    - pip install visbrain
```

Comment:
how come you don't need `radatools` in `travis`?

Reply:
> how come you don't need `radatools` in `travis`?

Yes it is true radatools should be installed in travis, but so far the nodes in interface/radatools are not tested yet as requiring the nipype engine to work, and for that the appveyor should be modified. 
Once the other PR is solved I will give a go at installing Radatools on travis